src/Client/Core/PurgeInstanceOptions.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask.Client;
+
+/// <summary>
+/// Options to purge an orchestration.
+/// </summary>
+public record PurgeInstanceOptions
```

Comment:
You can simplify a record type:

``` CSharp
public record PurgeInstanceOptions(bool Recursive = false);
```

Also, we need to keep recursive as default `false`. Otherwise, the default is a breaking behavior change from what we have today (which is non-recursive).

Reply:
Actually default value is intentionally kept as `true` here, but I agree it would be a breaking change. The thing is it is already set as true at other places like `durabletask-python` and even in `Dapr`, so setting it to false here would make it inconsistent. thoughts here @cgillum ?

Reply:
Okay, makes sense.

Reply:
Done